notebooks/240-named-entity-recognition/240-named-entity-recognition.ipynb

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
<p>not looks like that this section is needed, probably it should be renamed into prerequisites (or install dependencies) and imports of demo specific packages should be moved closer to place where it is used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
removed